pool/paymentmgr.go

```diff
@@ -544,8 +545,23 @@ func (pm *PaymentMgr) calculatePayments(ratios map[string]*big.Rat, source *Paym
 		}
 
 		paymentTotal += amt
-		payments = append(payments, NewPayment(account, source, amt, height,
-			estMaturity))
+
+		// The script size of the output is assumed to be the worst possible,
+		// which is txsizes.P2PKHOutputSize, to avoid a lower size estimation.
+		if txrules.IsDustAmount(amt, txsizes.P2PKHOutputSize,
+			txrules.DefaultRelayFeePerKb) {
+			// Since dust payments will cause the payout transaction to error
+			// and are also most likely to be generated by participating
+			// accounts contributing sporadic work to pool,  for these reasons
+			// dust payments will be forfeited by the accounts that generated
+			// them and be added to the pool fee payout. This intended
+			// to serve as a deterrent for contributinf sporadic work
+			// to the pool.
```

Comment:
```suggestion
			// Since dust payments will cause the payout transaction to error
			// and are also most likely to be generated by participating
			// accounts contributing sporadic work to pool,
			// dust payments will be forfeited by the accounts that generated
			// them and be added to the pool fee payout. This is intended
			// to serve as a deterrent for contributing sporadic work
			// to the pool.
```